metecho/api/models.py

```diff
@@ -451,11 +454,13 @@ def should_update_merged(self):
         return self.pr_is_merged
 
     def should_update_status(self):
-        return (
-            self.should_update_in_progress()
-            or self.should_update_review()
-            or self.should_update_merged()
-        )
+        if self.should_update_merged():
+            return self.status != PROJECT_STATUSES.Merged
+        elif self.should_update_review():
+            return self.status != PROJECT_STATUSES.Review
+        elif self.should_update_in_progress():
+            return self.status != PROJECT_STATUSES["In progress"]
+        return False
```

Comment:
@wlonk This change was to avoid spurious `PROJECT_UPDATED` events when the task name changes.